build.gradle

```diff
@@ -134,13 +134,19 @@ project(':iceberg-data') {
   dependencies {
     compile project(':iceberg-api')
     compile project(':iceberg-core')
+
     compileOnly project(':iceberg-parquet')
+    compileOnly project(':iceberg-orc')
+    compileOnly("org.apache.hadoop:hadoop-common") {
```

Comment:
Why was it necessary to add this hadoop-common dependency?

Reply:
Since `org.apache.hadoop:hadoop-common` is a non-transitive dependency used at compilation time on the other projects we need to add it here as ORC requires it for using `org.apache.hadoop.io.WritableComparable` on `HiveDecimalWritable`.

Reply:
I just thought that hadoop-client would pull it in transitively.